src/app/Shopping/Cart/Lens.tsx

```
@@ -0,0 +1,91 @@
+import styled from "styled-components/native";
+import { Font, color } from "../../../styles";
+import { Input } from "../../../components";
+import Map from "./Map";
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)